dev/remote/generate-test-env.js

```diff
@@ -5,10 +5,18 @@ const fse = require('fs-extra')
 const pkg = require('../../package.json')
 const automatedRegistration = require('./automated_registration')
 
-const cozyUrl = process.env.COZY_URL
+const cozyUrl = chooseCozyUrl(process.env.APPVEYOR_BUILD_NUMBER)
```

Comment:
This makes the script appveyor-dependent (while it was not necessarily).
But fixing AppVeyor has priority :)